dashboard/cookbook/hubspotforms.md

```diff
@@ -0,0 +1,123 @@
+# HubSpot Forms
+
+The proxy supports translation of HubSpot (or similar) forms via a combination of [project linking](../../menu/dashboard/linkprojects.html) and [JS translation](../../cookbook/jstranslation.html).
+
+**Method #1** marshals a combination of advanced proxy features. No change on the original server is necessary (which a frequent constraint). It's benefit is that it is entirely **hands-off** from the site maintainer's perspective.
+
+**Method #2** relies on HubSpot for localized forms and uses a proxy-compatible injected JS approach.
```

Comment:
@lsoos It think it would be great to mention here as well, that the Method #2 is simpler, but has the disadvantage, that there is a different form for each target languages -- so that a user, who doesn't mind that, won't get into reading and trying to understand Method #1 unnecessarily (it's tough to understand it, as I see it).